### DISCUSSION TITLE
IPv6 configuration? Not Possible or just using the wrong settings?

### DISCUSSION THREAD

**Discussion by KRGT-NL** [hassio-addons/addon-wireguard#153] [General]
Hi there!

I want to dual-stack my Wireguard VPN clients, but adding a link-local, like fd68:41bc:9e30:5b61::/64 network to the configuration it will not start and gives the error:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: WireGuard
 Fast, modern, secure VPN tunnel
-----------------------------------------------------------
 Add-on version: 0.5.1
 You are running the latest version of this add-on.
 System: Debian GNU/Linux 10 (buster)  (amd64 / qemux86-64)
 Home Assistant Core: 2021.9.5
 Home Assistant Supervisor: 2021.09.0
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] config.sh: executing... 
[cont-init.d] config.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[13:39:39] INFO: Starting WireGuard...
[#] ip link add wg0 type wireguard
[#] wg setconf wg0 /dev/fd/63
[#] ip -4 address add 172.27.66.1/32 dev wg0
[#] ip -6 address add fd68:41bc:9e30:5b61::1/64 dev wg0
RTNETLINK answers: Permission denied
[#] ip link delete dev wg0
```

Is there a way of getting this to work?

**Comment by JC-LB** [comment]
Same for me,
Don't know if this 2019 topic is still actual:
[https://github.com/hassio-addons/addon-wireguard/issues/25](https://github.com/hassio-addons/addon-wireguard/issues/25)

HASSIO docker "host network" provides IPv6. NodeRed Addon and Core are ok in v6.
But Wireguard is using "HASSIO" network.

I will try replace the add on by a standard WireGuard container managed using Portainer.
When using other container we sadly cannot use autoupdate tools such as wireguard, even while confuguring them to exclude managing HA containers.... Strange that this is impossible to disable in this open source environment.

Please move WG interface to "host" docker network so we can use IPv6 as we can do in RedNode or Core HTTPS.
